docs/extra_pages.dox

```diff
@@ -115,6 +115,146 @@
  * import polycap
  * \endcode
  *
+ * \page pyth_docs Python Docs
```

Comment:
Als je de Doxygen docs genereert met deze wijzigingen, zie je ze dan ook opduiken als je de index.html opent in je webbrowser? Indien niet (wat ik vermoed), zal je de `navindex` block in  `docs/DoxygenLayout.xml` moeten aanpassen door er een extra tab voor toe te voegen.

Reply:
Kvind niet meer hoe ik de docs kan genereren... Een gewone `make` of `make check` update de docs niet.

Reply:
Ge zit waarschijnlijk configure te runnen op een systeem zonder Doxygen. Zit ge bezig op een van de XMI computers?

Reply:
Ja, zit idd op XMI4 te werken (zoals meestal, op lvserver dacht ik issues te hebben met hdf5 packages enzo)

Reply:
Als ik u was zou ik eerder uw development lokaal doen met een VM... dan kunt ge een recente Linux distro installeren en root privileges hebben 😄